packages/ozone/src/mod-service/index.ts

```diff
@@ -317,6 +320,25 @@ export class ModerationService {
       }
     }
 
+    if (isAccountEvent(event)) {
+      meta.active = !!event.active
```

Comment:
If `active` is missing do we want to interpret it as though the account went inactive?  I wonder if we may want to omit it from `meta` altogether in this case.

Reply:
I guess the `!!` is confusing here but this is a rerouted from `#account` from the firehose and the `active` flag should be always set https://github.com/bluesky-social/atproto/blob/main/lexicons/com/atproto/sync/subscribeRepos.json#L133